src/kibana/components/doc_table/components/table_header.html

```diff
@@ -3,11 +3,12 @@
   <span ng-click="sort(indexPattern.timeFieldName)" tooltip="Sort by time">Time <i ng-class="headerClass(indexPattern.timeFieldName)"></i></span>
 </th>
 <th ng-repeat="name in columns">
-  <span ng-click="sort(name)" class="table-header-name" tooltip="{{tooltip(name)}}">
-    {{name | shortDots}} <i ng-class="headerClass(name)"></i>
+  <span class="table-header-name">
+    {{name | shortDots}} <i ng-class="headerClass(name)" ng-click="sort(name)" tooltip="{{tooltip(name)}}" tooltip-append-to-body="1"></i>
```

Comment:
I moved the click out to the sort icon and away from the entire header. There were enough icons up there that it was getting confusing.